Makefile.dapper

```diff
@@ -0,0 +1,19 @@
+# This Makefile contains the rules required to set up our
+# Dapper-based build environment; it can be copied as-is to
+# other projects (and needs to be copied, it can't be shared
+# via the Dapper image since it's needed to retrieve the image)
+
+.dapper:
+	@echo Downloading dapper
+	@curl -sL https://releases.rancher.com/dapper/latest/dapper-`uname -s`-`uname -m` > .dapper.tmp
+	@@chmod +x .dapper.tmp
+	@./.dapper.tmp -v
+	@mv .dapper.tmp .dapper
+
+%: .dapper
+	./.dapper -m bind $(MAKE) $@ $(MAKEFLAGS)
```

Comment:
```suggestion
	./.dapper -m bind make $@ $(MAKEFLAGS)
```

Please use "make" here directly, otherwise on other OSs  the host make will pass down the path for make into the container... and it will not work

Reply:
```suggestion
	+./.dapper -m bind make $@ $(MAKEFLAGS)
```

And if you’re specifying `make` directly, please use `+` (see my comments on Miguel’s PRs for details).

Reply:
Correct, I just saw it... thanks @skitt

Reply:
Thanks, added as commit!